Services/DataBaseService/Database/Models/DbUser.cs

```diff
@@ -1,16 +1,70 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 namespace DataBaseService.Database.Models
 {
-    public class DbUser
+    public class DbUser : IEquatable<DbUser>
     {
+        public static bool operator ==(DbUser first, DbUser second)

```

Comment:
Why you updated the class (and not only this one), will anyone need comparison anywhere apart from tests?